src/components/TextInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+function RenderTextField({ ...props }) {
+  const textProp = Object.assign({}, props);
+  delete textProp.meta;
+  delete textProp.input;
+
+  return (
+    <TextField
+      {...textProp}
```

Comment:
I think you can do:

```
{...props, meta: undefined, input:undefined}
```

And it would have the same effect as deleting, except be a bit terser.